CHANGELOG.asciidoc

```diff
@@ -33,6 +33,7 @@ There will always be one history file `${log_file}.1`.
 * Add <<config-log-format-sout>> and <<config-log-format-file>> with the options `PLAIN_TEXT` and `JSON`.
 The latter uses https://github.com/elastic/ecs-logging-java[ecs-logging-java] to format the logs.
 * Exposing <<config-classes-excluded-from-instrumentation>> config - {pull}1187[#1187]
+* Add support for naming transactions based on Grails controllers
```

Comment:
adding a version range in changelog would be great just to set user expectations.